pom.xml

```diff
@@ -14,35 +14,34 @@
     <name>document-signing-request-consumer</name>
     <description>Consumes Kafka messages from the sign-digital-document topic</description>
     <properties>
-        <java.version>11</java.version>
-        <spring-boot-dependencies.version>2.7.8</spring-boot-dependencies.version>
-        <maven.compiler.source>11</maven.compiler.source>
-        <maven.compiler.target>11</maven.compiler.target>
-
-        <cucumber.version>7.8.1</cucumber.version>
-        <environment-reader-library.version>1.3.8</environment-reader-library.version>
-        <junit.version>5.9.1</junit.version>
-        <kafka-models.version>1.0.40</kafka-models.version>
-        <ch-kafka.version>1.4.8</ch-kafka.version>
-        <mockito.version>4.8.0</mockito.version>
-        <spring-boot-dependencies.version>2.7.8</spring-boot-dependencies.version>
-        <private-api-sdk-java.version>2.0.322</private-api-sdk-java.version>
-        <api-sdk-manager-java-library.version>1.0.6</api-sdk-manager-java-library.version>
-
+        <java.version>21</java.version>
+        <spring-boot-dependencies.version>3.3.5</spring-boot-dependencies.version>
+        <maven.compiler.source>21</maven.compiler.source>
+        <maven.compiler.target>21</maven.compiler.target>
+        <avro.version>1.12.0</avro.version>
+        <cucumber.version>7.20.1</cucumber.version>
+        <environment-reader-library.version>3.0.1</environment-reader-library.version>
+        <junit.version>5.10.3</junit.version>
+        <kafka-models.version>3.0.8</kafka-models.version>
+        <ch-kafka.version>3.0.3</ch-kafka.version>
+        <mockito.version>5.14.2</mockito.version>
+        <private-api-sdk-java.version>4.0.225</private-api-sdk-java.version>
+        <api-sdk-manager-java-library.version>3.0.6</api-sdk-manager-java-library.version>
+        <build-helper-maven-plugin>3.6.0</build-helper-maven-plugin>
+        <maven-failsafe-plugin.version>3.5.1</maven-failsafe-plugin.version>
+        <maven-surefire-plugin.version>3.0.0-M7</maven-surefire-plugin.version>
+        <system-stubs.version>2.1.7</system-stubs.version>
         <start-class>uk.gov.companieshouse.documentsigningrequestconsumer.DocumentSigningRequestConsumerApplication</start-class>
-        <spring-boot-maven-plugin.version>2.5.5</spring-boot-maven-plugin.version>
-        <!-- system-rules: 1.17.2 is the latest version that works with JUnit 5.
-            See https://github.com/stefanbirkner/system-rules/issues/70 -->
-        <system-rules-version>1.17.2</system-rules-version>
+        <spring-boot-maven-plugin.version>3.3.5</spring-boot-maven-plugin.version>
+        <system-rules-version>1.19.0</system-rules-version>
+        <jetty-ee10-webapp.version>12.1.0.alpha0</jetty-ee10-webapp.version>
```

Comment:
[I would avoid using an alpha version too if possible](https://mvnrepository.com/artifact/org.eclipse.jetty.ee10/jetty-ee10-webapp/12.0.15)

Reply:
its well noted, its the latest and i will check again before committing else use the version before this one